src/opcodes.js

```diff
@@ -356,11 +351,16 @@ module.exports = new Map([
 
   }],
 
-  ['OP_INVALID[OPCODE', function (context) {
-    throw Error(`Invalid opcode ${context.script.at(this.programCounter).toString(16)}`)
+  ['OP_INVALIDOPCODE', function (context) {
+    throw Error(`Invalid opcode ${context.script.at(context.programCounter).toString(16)}`)
   }]
 ])
 
+function opFalse (context) {
+  context.stack.push(Buffer.from('00', 'hex'))
```

Comment:
```suggestion
  context.stack.push(Buffer.alloc(0))
```

According to the wiki:

> An empty array of bytes is pushed onto the stack. (This is not a no-op: an item is added to the stack.)

I believe the data is supposed to be empty.